snap/snapcraft.yaml

```
@@ -38,6 +41,7 @@ parts:
   jenkins:
     plugin: nil
     source: https://github.com/jenkinsci/jenkins.git
+    source-tag: $SNAPCRAFT_PROJECT_VERSION    
```

The tags in the upstream repo are prefixed with `jenkins-`

```suggestion
    source-tag: jenkins-$SNAPCRAFT_PROJECT_VERSION    
```

See https://github.com/jenkinsci/jenkins/tags

---

Thanks @merlijn-sebrechts - I appreciate your patience w/ me as I learn these workflows and their ins 'n outs. I think, maybe, this time, it's good.

---

Don't beat yourself up about it! Everyone makes mistakes.